tests/go/GofmtBearTest.py

```diff
@@ -5,3 +5,11 @@
     GofmtBear,
     ('package main\n\nfunc main() {\n\treturn 1\n}',),
     ('package main\nfunc main() {\n\treturn 1\n}',))
+
+GofmtBearWithSettings = verify_local_bear(
+    GofmtBear,
```

Comment:
And here the replaced `GofmtBear` variable content as described above is given to `verify_local_bear`, which expects a bear class and not a test case class. And better name the assignment `GofmtBearWithSettingsTest = ...`